Userland/Libraries/LibSemVer/SemVer.cpp

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright (c) 2023, Gurkirat Singh <tbhaxor@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <AK/Find.h>
+#include <AK/Function.h>
+#include <AK/GenericLexer.h>
+#include <AK/ReverseIterator.h>
+#include <AK/StringBuilder.h>
+#include <LibRegex/Regex.h>
+#include <LibRegex/RegexParser.h>
+#include <LibSemVer/SemVer.h>
+
+namespace SemVer {
+String SemVer::suffix() const
+{
+    StringBuilder sb;
+    if (m_prerelease_identifiers.size() > 0)
+        sb.appendff("-{}", prerelease());
+    if (m_buildmetadata.code_points().length() > 0)
+        sb.appendff("+{}", buildmetadata());
+    return MUST(sb.to_string());
+}
+
+String SemVer::to_string() const
+{
+    return MUST(String::formatted("{}{}{}{}{}{}", m_major, m_number_separator, m_minor, m_number_separator, m_patch, suffix()));
```

Comment:
Same thing here regarding `MUST`.

Reply:
Could you help me recall? Do I have to use ErrorOr here?